src/layout/landingPage/Footer.js

```diff
@@ -1,195 +1,82 @@
 import React from 'react';
-import {
-  Flex,
-  FlexItem,
-  Split,
-  SplitItem,
-  TextContent,
-  Text,
-} from '@patternfly/react-core'; //eslint-disable-line
-import {
-  ArrowRightIcon,
-  BuilderImageIcon,
-  CloudSecurityIcon,
-  CloudTenantIcon,
-  ConnectedIcon,
-  ExternalLinkAltIcon,
-} from '@patternfly/react-icons';
+import { Flex } from '@patternfly/react-core';
+import configTryLearnRenderer from '../../components/app-content-renderer/config-try-learn-renderer';
 
-import IconInsights from '../../components/icon-insights';
-import IconAnsible from '../../components/icon-ansible';
+const mockData = [
```

Comment:
No need to use mockData, you can pass these values directly to https://github.com/RedHatInsights/landing-page-frontend/blob/master/src/utils/config.json#L84